cmd/subnetcmd/join.go

```diff
@@ -58,7 +64,10 @@ This command currently only supports subnets deployed on the Fuji testnet.`,
 	}
 	cmd.Flags().StringVar(&avagoConfigPath, "avalanchego-config", "", "file path of the avalanchego config file")
 	cmd.Flags().StringVar(&pluginDir, "plugin-dir", "", "file path of avalanchego's plugin directory")
+	cmd.Flags().StringVar(&networkStr, "network", "", "avalanche network where the validator will be joining (either `fuji` or `mainnet`)")
```

Comment:
Same global decision I mentioned in the other PR

Reply:
Fixed.